w3af/core/controllers/chrome/instrumented/main.py

```diff
@@ -589,19 +608,13 @@ def js_runtime_evaluate(self, expression, timeout=5):
                                                    timeout=timeout)
 
         # This is a rare case where the DOM is not present
-        if result is None:
-            return None
-
-        if 'result' not in result:
-            return None
-
-        if 'result' not in result['result']:
-            return None
-
-        if 'value' not in result['result']['result']:
-            return None
-
-        return result['result']['result']['value']
+        runtime_exception = result.get('result', {}).get('exceptionDetails')
+        if runtime_exception:
+            raise ChromeScriptRuntimeException(
```

Comment:
This is new! Are you sure it is handled in all calls to this method? In the past we were returning `None` and now we raise an exception. Make sure to search all calls for this method and modify error handling appropiately.

Reply:
Yeah, I have added few more fixes in this PR.
This works for sure. It's additional (new) exception which is raised when underlying script throws internal JS exception. We still return `None` in other cases here https://github.com/andresriancho/w3af/pull/18552/files/8d520bc067455089a9fccac9c395f9114f521e8a#diff-4a954d4e5c6123fad237c66f5399b9dfR617